docs/development/design/new-sphinx-guides.rst

```diff
@@ -0,0 +1,186 @@
+
+   2. Creating the project layout
+
+      -  Run the sphinx-quickstart wizard: separate source and build
+         (``y``), project name, author name, and go
+      -  Check that the correct files are created (yay!)
+
+   3. Converting our documentation to local HTML
+
+      -  Build the HTML output using
+         ``sphinx-build -b -W html doc doc/_build/html``  [1]_
+      -  Navigate to ``doc/_build/html`` and launch an HTTP server
+         (``python -m http.server``)
+      -  Open http://localhost:8000 in a web browser, and see the HTML
+         documentation (yay!)
+
+   4. Making a change and regenerating the documentation
+
+      -  Add a sentence to ``index.rst`` (no need to explain in detail
+         what reST is at this stage)
+      -  Rebuild HTML ``sphinx-build -W -b html ...``
+      -  Observe that the HTML docs have changed (yay!) (we will have to
+         do this every time we change something)
+
+   5. Simplify documentation building by using Make
+
+      -  Install Make (and its Windows counterpart)
+      -  Make another trivial change
+      -  Build HTML doing ``cd doc && make html``
+      -  Observe that the HTML docs have changed (yay!)
+
+   6. Converting our documentation to PDF
+
+      -  Install TeX Live to create PDF output (should work on every
+         operative system)
+      -  Build LaTeX using ``make latexpdf``
+      -  See that the PDF appeared (yay!)
+
+   7. Appendix: PDF without LaTeX using rinoh (beta)
+
+3. Customizing Sphinx configuration
+
+   1. Changing the HTML theme
+
+      -  Install https://pypi.org/project/furo/
+      -  Change the ``html_theme`` in ``conf.py``
+      -  Rebuild the HTML documentation and observe that the theme has
+         changed (yay!)
+
+   2. Changing the PDF appearance
+
+      -  Add a ``latex_theme`` and set it to ``howto``
+      -  Rebuild ``make latexpdf``
+      -  Check that the appearance changed (yay!)
+
+   3. Enable an extension
+
+      -  Add a string to the ``extensions`` list in ``conf.py`` for
+         ``sphinx.ext.duration``
+      -  Rebuild the HTML docs ``make html`` and notice that now the
+         times are printed (yay!)
+
+4. Writing narrative documentation with Sphinx
+
+   -  First focus on ``index.rst``, gently introducing the learner to
+      reST and mentioning Semantic Line Breaks.
+   -  Then add another ``.rst`` file to teach how ``toctree`` works.
+   -  Then enable Markdown adding ``myst_parser`` to the extensions and
+      authoring a Markdown file, to show that both can exist at the same
+      time.
+   -  Then continue introducing elements of the syntax to add pictures,
+      cross-references, and the like.
+
+5. Using Jupyter notebooks inside Sphinx
+6. Describing code in Sphinx
+
+   -  Explain the Python domain as part of narrative documentation to
+      interleave code with text, include doctests, and justify the
+      usefulness of the next section.
+
+7. Autogenerating documentation from code in Sphinx
+
```

Comment:
We should include an RTD section here, or at least a pointer to an RTD continuation of the tutorial? It seems like this would be the natural end-point with deploying.

Reply:
Absolutely!

Reply:
Added section on deployment, deferring to the actual RTD tutorial which I still have to propose a TOC for.